day22/permutations.py

```diff
@@ -0,0 +1,20 @@
+def permutations(arr):
+    allPermutations = list()
+
+    if len(arr) <= 1:
```

Comment:
i know that 0 and 1 are good base cases, but maybe it would make the conceptual algorithm even better if you only did 0 as the base case?